mitodl/mitxonline
# Updates course run and program certificate models to limit choices just to certificate pages in admin

#### Pre-Flight checklist


#### What are the relevant tickets?

Closes #1283 

#### What's this PR do?

Updates the program and course run certificate models' certificate page foreign keys to limit their choices to actual certificate pages, rather than all CMS pages.

#### How should this be manually tested?

1. Ensure you have certificate pages set up properly.
2. Apply the migrations.
3. In Django Admin, edit or create either a program or a course run certificate. The options in the list for Certificate page revision should only be certificate pages. (Prior to this, all CMS pages were listed; you could set the cert page to be the MITx Online homepage, for instance.) 
4. Repeat step 3 with the courseware object type you haven't yet tested.
